src/core/player.h

```diff
@@ -177,10 +179,10 @@ class Player : public PlayerInterface {
   void TrackEnded();
   // Play the next item on the playlist - disregarding radio stations like
   // last.fm that might have more tracks.
-  void NextItem(Engine::TrackChangeFlags change);
+  void NextItem(Engine::TrackChangeFlags change, bool next_album = false);
```

Comment:
Can you make `next_album` an enum for readability?

Reply:
Per my conversation with Jim I'm going to create a new function that looks for the next album/track to play so this may become OBE.

Reply:
Eventually I decided to leave the code mostly as-is rather than writing a completely new method - so I applied your suggestion and committed it as a separate commit.